### ISSUE TITLE
Decide on dependency updating schedule

### ISSUE THREAD

**Issue by yodasw16** [sparkdesignsystem/spark-design-system#4]
Come up with a regular schedule for making sure our dependencies are up to date.

**Comment by RCopeland** [comment]
I think we should have a dep update step before each release. Thoughts?

**Comment by yodasw16** [comment]
Yes, I think that makes sense. Make sure everything is as up to date as possible at the time of release.
